quickstart/201-private-link-sql-database/provider.tf

```diff
@@ -16,5 +16,9 @@ terraform {
 }
 
 provider "azurerm" {
```

Comment:
@asudbring It might just be the way it's viewing on my PC, but the indentation looks off. Please make sure to run "terraform fmt". Thanks!

Reply:
will do

Reply:
ran terraform fmt and pushed changes